src/components/SliderArrows.js

```diff
@@ -146,3 +146,132 @@ export const AltLeftArrow = props => {
     </div>
   )
 }
+
+
+
+
+
+export const RevealRightArrow = props => {
+  return (
+    <div
+      className='nextArrow'
+      onClick={props.clickAction}
+      onMouseEnter={props.over}
+      onMouseMove={props.over}
+      onMouseLeave={props.out}
+      style={{
+        position: 'absolute',
+        top: '0',
+        right: '0',
+        width: '40%',
+        height: '100%',
+        zIndex: '101',
+        cursor: 'pointer',
+        WebkitTapHighlightColor: 'rgba(0,0,0,0)'
+      }}>
+      <div className='right-arrow-wrapper'>
+        <svg className={`revealArrow ${props.clickedArrow ? 'clicked' : ''}`} width="17" height="22" viewBox="0 0 17 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Maybe consider using this svg as a background and then rotating/flipping it for the other side. That way only one asset is used.